.github/workflows/release_win.yml

```diff
@@ -1,9 +1,10 @@
 name: WindowsRelease
 
 on:
-  push:
-    branches:
-      - rel-*
+   push:
+    branches: [ master, rel-* ]
+   pull_request:
+    branches: [ master, rel-* ]
```

Comment:
Since the release was done, do we still need to run the release-CI for master branch?

Reply:
we should run the release CI for master branch pushes (not pull requests)... this is a good way of making sure the CI pipelines are running fine

Reply:
Make sense. Updated. Thanks